### ISSUE TITLE
Release Che 7.25.2

### ISSUE THREAD

**Issue by nickboldt** [eclipse-che/che#18926]
### List of pending issues / PRs
* [x] BLOCKER: Openvsx dropped some extensions #18933
* [x] Release Che 7.25.1 #18911
* [x] NEW workflow: Include jwtproxy and KIP in Che release - https://github.com/eclipse/che-release/pull/24 https://github.com/eclipse/che/issues/17743 https://github.com/eclipse/che/issues/17742
* [x] update antora playbook for each Che release - https://github.com/eclipse/che-docs/pull/1823 
* [x] NEW workflow: trigger che-doc release when che-operator is done - https://github.com/eclipse/che-operator/pull/657 
  * [x] merge eclipse/che-docs#1836 and eclipse/che-operator#660; verify che-operator action triggers che-docs action (correct tokens used)
  * [ ] verify antora playbook PR submitted for che-docs master branch, with valid content
* [x] NEW workflow: che server release as GH action - https://github.com/eclipse/che/issues/18780

### Release status

In series, the following will be released via [che-release](https://github.com/eclipse/che-release/blob/master/cico_release.sh):

* che-theia, che-machine-exec, che-devfile registry, 
* che-plugin-registry (once che-theia and machine-exec are done)
* che-parent, che-dashboard, che-workspace-loader, and che (server assembly), including release to [Maven Central](https://github.com/eclipse/che/issues/16043)
* che-operator

Then, these steps will be done, which still [require some manual intervention](https://github.com/eclipse/che-release/blob/master/README.md#phase-2---manual-steps) (PR verification is not yet automated):

* generation of OperatorHub PRs
* chectl

- [x] che-theia, che-machine-exec, che-devfile-registry, che-plugin-registry
- [x] che-parent, che-dashboard, che-workspace-loader, che
- [x] che-operator _(depends on all of the above)_
- [ ] chectl _(depends on che-operator)_
- [ ] [Che community operator PRs](https://github.com/operator-framework/community-operators/pulls?q=%22Update+eclipse-che+operator%22+is%3Aopen) _(depends on che-operator)_

If this is a .0 release:

- [ ] [send reminder to TLs to triage unresolved issues](https://github.com/eclipse/che/issues/17579)
- [ ] complete current milestone
  - [ ] move incomplete *deferred* issues to backlog
  - [ ] move incomplete *WIP* issues to next milestone
  - [ ] close completed issues
  - [ ] close milestone

| Owner | Process | Script | CI | Artifact(s) + Container(s) |
| --- | --- | --- | --- | --- |
| @mkuznyetsov  | [che-theia](https://github.com/eclipse/che-theia/blob/master/RELEASE.md) | [cico_release.sh](https://github.com/eclipse/che-release/blob/master/cico_release.sh) | [centos](https://ci.centos.org/job/devtools-che-theia-che-release/) | [`eclipse/che-theia`](https://quay.io/eclipse/che-theia) |
| @mkuznyetsov  | [che-machine-exec](https://github.com/eclipse/che-machine-exec/blob/master/RELEASE.md) | [cico_release.sh](https://github.com/eclipse/che-release/blob/master/cico_release.sh) | [centos](https://ci.centos.org/job/devtools-che-machine-exec-release/) | [`eclipse/che-machine-exec`](https://quay.io/eclipse/che-machine-exec)| 
| @mkuznyetsov  | [che-devfile-registry](https://github.com/eclipse/che-devfile-registry/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-devfile-registry/blob/master/make-release.sh) | [GH action](https://github.com/eclipse/che-devfile-registry/actions?query=workflow%3A%22Release+Che+Devfile+Registry%22) | [`eclipse/che-devfile-registry`](https://quay.io/eclipse/che-devfile-registry)| 
| @mkuznyetsov  | [che-plugin-registry](https://github.com/eclipse/che-plugin-registry/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-plugin-registry/blob/master/make-release.sh) | [GH action](https://github.com/eclipse/che-plugin-registry/actions?query=workflow%3A%22Release+Che+Plugin+Registry%22) | [`eclipse/che-plugin-registry`](https://quay.io/eclipse/che-plugin-registry)| 
| @mkuznyetsov  | [che-parent](https://github.com/eclipse/che-parent/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-parent/blob/master/make-release.sh) | [centos](https://ci.centos.org/job/devtools-che-release-che-release) | [che-parent](https://search.maven.org/search?q=a:che-parent) |
| @mkuznyetsov  | [che-dashboard](https://github.com/eclipse/che-dashboard/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-dashboard/blob/master/make-release.sh) | [GH action](https://github.com/eclipse/che-dashboard/actions?query=workflow%3A%22Release+Che+Dashboard%22) | [`che-dashboard`](https://quay.io/repository/eclipse/che-dashboard?tag=next&tab=tags) |
| @mkuznyetsov  | [che-workspace-loader](https://github.com/eclipse/che-workspace-loader/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-workspace-loader/blob/master/make-release.sh) | [GH action](https://github.com/eclipse/che-workspace-loader/actions?query=workflow%3A%22Release+Che+Workspace+Loader%22) | [`che-workspace-loader`](https://quay.io/repository/eclipse/che-workspace-loader?tag=next&tab=tags) |
| @mkuznyetsov  | [che](https://github.com/eclipse/che/blob/master/RELEASE.md) | [cico_release.sh](https://github.com/eclipse/che-release/blob/master/cico_release.sh) | [centos](https://ci.centos.org/job/devtools-che-release-che-release) | [che.core](https://search.maven.org/search?q=che.core),<br/> [`eclipse/che-server`](https://quay.io/eclipse/che-server),<br/>[`eclipse/che-endpoint-watcher`](https://quay.io/eclipse/che-endpoint-watcher),<br/> [`eclipse/che-keycloak`](https://quay.io/eclipse/che-keycloak),<br/> [`eclipse/che-postgres`](https://quay.io/eclipse/che-postgres),<br/> [`eclipse/che-server`](https://quay.io/eclipse/che-server),<br/> [`eclipse/che-e2e`](https://quay.io/eclipse/che-e2e) |
| @mkuznyetsov| [che-operator](https://github.com/eclipse/che-operator/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-operator/blob/master/make-release.sh) | [GH action](https://github.com/eclipse/che-operator/actions?query=workflow%3A%22Release+Che+Operator%22) | [`eclipse/che-operator`](https://quay.io/eclipse/che-operator)| 
| @mkuznyetsov| [chectl](https://github.com/che-incubator/chectl/blob/master/RELEASE.md) | [make-release.sh](https://github.com/che-incubator/chectl/blob/master/make-release.sh) | [GH action](https://github.com/che-incubator/chectl/actions) | [chectl releases](https://github.com/che-incubator/chectl/releases)

**Comment by ibuziuk** [comment]
@nickboldt @mkuznyetsov could you please let me know once artifacts are available - waiting for the z-stream since for Hosted Che update

**Comment by mkuznyetsov** [comment]
looks like che-theia build is unavailable right now, from what looks like openvsx related issues:
https://github.com/eclipse/che/issues/18960
https://github.com/eclipse-theia/theia/issues/9014#issuecomment-772633182

@ibuziuk  I will have to wait to get unblocked on this, since che-server comes after theia&registries being built

**Comment by azatsarynnyy** [comment]
@mkuznyetsov 
The open-vsx-related issue is fixed.
Che-Theia build is stabilized now.
https://github.com/eclipse/che-theia/pull/979

**Comment by mkuznyetsov** [comment]
@ibuziuk FYI right now stuck che-server release, having timeouts during deploy just like the last time, when there were issues with sonatype. (and it looks like they're back https://twitter.com/sonatype_ops )

**Comment by ibuziuk** [comment]
@mkuznyetsov thanks, please let me know once the artifacts are available - Ideally would be great to promote '7.25.2' on staging before the weekend

**Comment by mkuznyetsov** [comment]
after several hours of retrying and waiting for artifacts to appear on Central, Che Server is now release, now on to operator PR testing

**Comment by mkuznyetsov** [comment]
release is complete, community operator PRs are merged
